frontend/apps/core-app/tsconfig.json

```diff
@@ -2,10 +2,14 @@
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
     "strict": true,
-    "paths": {
-      "core-api-client": ["../../packages/core-api-client/src"],
-      "core-auth": ["../../packages/core-auth/src"],
-      "core-design-system": ["../../packages/design-system/src"],
-    }
-  }
+    "allowSyntheticDefaultImports": true,
```

Comment:
this was changed when I ran yarn test ...

Reply:
hmm strange, I'll try running it myself and seeing what happens. I know `react-scripts` changes `tsconfig.json` but I wouldn't expect expo or jest to.